buildbot/osuosl/master/config/status.py

```diff
@@ -407,14 +407,14 @@ def getReporters():
             fromaddr = "llvm.buildmaster@lab.llvm.org",
             sendToInterestedUsers = False,
             messageFormatter = LLVMInformativeMailNotifier,
-            extraRecipients = ["llvm-premerge-buildbots@google.com"],
+            extraRecipients = ["llvm-premerge-buildbots@google.com", "joker.eph@gmail.com"],
             mode = "failing",
             builders = ["premerge-monolithic-windows", "premerge-monolithic-linux"]),
         reporters.MailNotifier(
             fromaddr = "llvm.buildmaster@lab.llvm.org",
             sendToInterestedUsers = False,
             messageFormatter = LLVMInformativeMailNotifier,
-            extraRecipients = ["szakharin@nvidia.com"],
+            extraRecipients = ["szakharin@nvidia.com""],
```

Comment:
Double quotation marks at the end of the line looks like a mistype. Did you mean adding yourself to the list of extra recipients?

Reply:
I added myself here and realized it was the wrong bot and didn't undo it properly. Thanks for catching!

I fixed it now.